rb_tree.py

```diff
@@ -35,29 +38,39 @@ def __eq__(self, other):
         if self.parent is None or other.parent is None:
             parents_are_same = self.parent is None and other.parent is None
         else:
-            parents_are_same = self.parent.value == other.parent.value and self.parent.color == other.parent.color
-        return self.value == other.value and self.color == other.color and parents_are_same
+            parents_are_same = self.parent.value == (
+             other.parent.value and self.parent.color) == other.parent.color
+
+        return (self.value == other.value and
```

Comment:
Could we please not change unnecessary stuff?

Reply:
It was changed in consequence of the PEP8 request. It exceeds the number of characters per line.